### ISSUE TITLE
Suggestion: Allow option to filter on routes

### ISSUE THREAD

**Issue by sebmor** [jeneser/vue-scroll-behavior#15]
There is already an option to ignore routes, but in the many cases where you only have one or a few pages that need the persist their scroll, it's often easier to only select these instead of ignoring all the rest.

I suggest a filter kind of option:

```
Vue.use(vueScrollBehavior, {
  filter: [/\/boo/, /\/zoo/],    // only apply on these routes,
})

**Comment by jeneser** [comment]
Right, we just need a `filter` option to match our routes. I marked it as a new feature for PR.

**Comment by tchret** [comment]
must have
